src/solvers/dgmulti/dg_parabolic.jl

```diff
@@ -1,8 +1,7 @@
 # version for standard (e.g., non-entropy stable or flux differencing) schemes
 function create_cache_parabolic(mesh::DGMultiMesh,
```

Comment:
Is this still used in this PR?

Reply:
Yes, for the DGMulti mesh this is really needed